### PR TITLE
Added count comparison to clickhouse export

### PR DESCRIPTION
**Pull request by StanislavZoldak**

### CODE REVIEW

File: src/__myproject__/_export/clickhouse_export.py [hunk 1]
```diff
@@ -119,3 +119,14 @@ def rename_tables(table_names: dict, logger: Logger):
...
 
     logger.info(f"Export successful, table saved as {table_names['main']}.")
+
+# COMMAND ----------
+
+@dp.notebook_function(features_to_export_with_conversions, get_table_names)
+def compare_counts(df: DataFrame, table_names: dict, logger: Logger):
+    fs_count = df.count()
+    clickhouse_count = execute_clickhouse_query(f"SELECT COUNT(*) from {table_names['main']}")[0]["count()"]
+    if int(fs_count) == int(clickhouse_count):
```
Comment (ds-lukaslangr) [inline src/__myproject__/_export/clickhouse_export.py:129]: they are not ints?

File: src/__myproject__/_export/clickhouse_export.py [hunk 2]
```diff
@@ -119,3 +119,14 @@ def rename_tables(table_names: dict, logger: Logger):
...
+        logger.info(f"Featurestore row count equals to Clickhouse export row count ({fs_count})")
+    else:
+        logger.warning(f"Featurestore row count is not equal to export row count. Featurestore: {fs_count} | Clickhouse: {clickhouse_count}")
```
Comment (ds-lukaslangr) [inline src/__myproject__/_export/clickhouse_export.py:132]: do we maybe wanna throw an Exception if it doesn't add up?

File: src/__myproject__/_export/clickhouse_export.py (hunk 1 above)
Comment (StanislavZoldak) [reply src/__myproject__/_export/clickhouse_export.py:129]: One of them wasn't.

Comment (StanislavZoldak) [reply src/__myproject__/_export/clickhouse_export.py]: Removed fs_count casting and moved clickhouse_count casting up.

**Review by ds-lukaslangr — APPROVED** [review]

**Pull request merged**